content/en/docs/collector/internal-telemetry.md

```diff
@@ -113,3 +155,149 @@ journalctl | grep otelcol | grep Error
 ```
 
 {{% /tab %}} {{< /tabpane >}}
+
+## Types of internal observability
+
+The OpenTelemetry Collector aims to be a model of observable service by clearly
+exposing its own operational metrics. Additionally, it collects host resource
+metrics that can help you understand if problems are caused by a different
+process on the same host. Specific components of the Collector can also emit
+their own custom telemetry. In this section, you can explore the different types
+of observability emitted by the Collector itself.
+
+### Values observable with internal metrics
+
+The Collector emits internal metrics for the following **current values**:
+
+- Resource consumption, including CPU, memory, and I/O.
+
+- Data reception rate, broken down by receiver.
+
+- Data export rate, broken down by exporters.
+
+- Data drop rate due to throttling, broken down by data type.
+
+- Data drop rate due to invalid data received, broken down by data type.
+
+- Throttling state, including Not Throttled, Throttled by Downstream, and
+  Internally Saturated.
+
+- Incoming connection count, broken down by receiver.
+
+- Incoming connection rate showing new connections per second, broken down by
+  receiver.
+
+- In-memory queue size in bytes and in units.
+
+- Persistent queue size.
+
+- End-to-end latency from receiver input to exporter output.
+
+- Latency broken down by pipeline elements, including exporter network roundtrip
+  latency for request/response protocols.
+
+{{% alert title="Important" color="warning" %}}
+
+- Rate values reflect the average rate of the last 10 seconds. They are exposed
+  in bytes/sec and units/sec (for example, spans/sec).
+- Measurements in bytes might be difficult and expensive to obtain. They should
+  be used with caution.
+
+{{% /alert %}}
+
+The Collector also emits internal metrics for these **cumulative values**:
+
+- Total received data, broken down by receivers.
+
+- Total exported data, broken down by exporters.
+
+- Total dropped data due to throttling, broken down by data type.
+
+- Total dropped data due to invalid data received, broken down by data type.
+
+- Total incoming connection count, broken down by receiver.
+
+- Uptime since start.
+
+### List of internal metrics
+
+The following table identifies each internal metric by name and description.
+Each metric is also categorized by instrumentation type and level of verbosity.
```

Comment:
This is a large table, but I'm ok leaving it as is for now -- even if readers need to scroll left-right on narrow displays.

A few comments and questions:

(1) Not all entries have a specified verbosity level / threshold. From a comment thread elsewhere, I think that those entries are `basic`. Let's make that explicit.

(2) Where is this list coming from? How quickly will it age? Will it be a maintenance burden? Can we alleviate any of this issues hinted at by the previous questions.

Reply:
> (1) Not all entries have a specified verbosity level / threshold. From a comment thread elsewhere, I think that those entries are basic. Let's make that explicit.

I'm working on that now. I was waiting for confirmation from the Collector folks that all the remaining metrics were, in fact, emitted at the `basic` level. 

> (2) Where is this list coming from? How quickly will it age? Will it be a maintenance burden? Can we alleviate any of this issues hinted at by the previous questions.

I used a grep command to find the metrics in the Collector core repository. I have no idea how often the list of metrics changes. The @open-telemetry/collector-approvers would know better.

Reply:
Cool. Maybe, at a minimum add a (doc-only) comment to that effect and even consider including the grep command(s) for reference?

Reply:
Sure. Added a comment in 12399b4.